### ISSUE TITLE
something is fishy with bitmapdata.copypixels on iOS since version 1.4.0

### ISSUE THREAD

**Issue by jgranick** [openfl/openfl-native#211]
#211 Issue by __stahlratte__,

I upgraded to 1.4.0 today and my pixels aren't copied right on iOS anymore. Android and Flash works fine.
Please check if it's working right in you're test cases. If it is, I can put together an example of what I am doing next week. Right now I'm to busy and the code is under an NDA sadly.

**Comment by jgranick** [comment]
Comment by __jgranick__:

In previous versions, we maintained a consistent pixel format at all times, but required swizzling of textures when it went to the GPU, depending on the system. Now, the textures can work with GPU-specific pixel orders.

In my tests, everything was consistent except for bitmapData.encode, which I addressed, but I did not encounter other problems, although this may have been in testing different platforms. iOS has a different color order.

Would it be hard to whip up a little test case? "lime create openfl:project CopyPixelBug" then try something small to see the same result you're getting now. Thank you!

**Comment by jgranick** [comment]
Comment by __stahlratte__:

I finally got the time to create an example project. it's located @ https://github.com/stahlratte/CopyPixelBugIOS
i just stripped the actual project down to the part that made the problems.
It's a slider thingy with a magnifying effect. With the real assets for the client it looks much nicer. ;)
it works flawlessly on all targets. only on ios i had to revert to the old openfl libs to get it working. i documented the haxlib setup i used in the README.md
I only tested this buggy version on an iPad 2 running iOS 6.1.2 and switched back to the old version immediatly when i noticed the problem, to get the project ready in time.
Don't know if there would be differences between hardware/os combos.

Oh the visual indicator for the bug is that the magnified bit of the slider isn't moving with openfl 1.4.0
